elastic_apm.permissions.yml

```diff
@@ -0,0 +1,2 @@
+administer elastic apm:
+  title: 'Administer Elastic APM'
```

Comment:
**CHANGE REQUEST**

The convention is to use the module's machine when defining a permission for configuring the module sitewide i.e. `administer elastic_apm` with an underscore.